hosting/setup-guides/linux.md

```diff
@@ -99,62 +88,92 @@ Address addr:333d10486632f11c4c5b907c2e45d31478522dec525649712697404b4253e92ea5a
 
 ## Setting up a systemd service
 
-Now that you have a recovery phrase, we will create a `systemd` service to run `hostd` on startup.
+Now that you have a recovery phrase, we will create a new system user and `systemd` service to securely run `hostd` on startup.
+
+First, we will create a new system user with `useradd` and disable the creation of a home directory. This is a security precaution which will isolate `hostd` from any unauthorized access to our system. We will then use `usermod` to lock the account and prevent anyone from logging in under the account.
+
+```
+sudo useradd -M hostd
+sudo usermod -L hostd
+```
 
-First, create a new environment file. Where you store the file is up to you, but you will need to know the path later. For this guide, we will store it in our home directory `/home/ubuntu/hostd.env`
+Now we will create a new folder under `/var/lib/` titled `hostd` and give our new system and set the approriate permissions. This folder will be utilized specifically to store data related to the `hostd` software. Open the Terminal Emulator and run the following commands:
 
 ```
-sudo nano /home/ubuntu/hostd.env
+sudo mkdir /var/lib/hostd
+sudo chown hostd:hostd /var/lib/hostd
+sudo chmod o-rwx /var/lib/hostd
 ```
 
-Now, modify the file to add your wallet seed and API password. The password is used to unlock the `hostd` UI, it should be something secure and easy to remember. The recovery phrase is the 12-word phrase you generated in the previous step. Type it carefully, with one space between each word, or copy it from the previous step.
+Next, create a file name `hostd.yml` file under `/var/lib/hostd/`
+
+```
+sudo nano /var/lib/hostd/hostd.yml
+```
+
+Now, modify the file to add your wallet seed and API password. The recovery phrase is the 12-word phrase you generated in the previous step. Type it carefully, with one space between each word, or copy it from the previous step. The password is used to unlock the `hostd` UI; it should be something secure and easy to remember.
+
+{% hint style="warning" %}
+The port `9980` is `hostd`'s default operating port and should not need to be changed unless you require a custom configuration for your network.
+{% endhint %}
 
 {% tabs %}
 {% tab title="Mainnet" %}
 ```
```

Comment:
Make sure all of the code blocks have language formatting

Reply:
Okay, I didn't see any on the other ones, and what was there was inconsistent, which is why I removed those 2. I'll fix them.